flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -178,21 +180,9 @@ function cleanup {
     stop_kubernetes
 }
 
-function setConsoleLogging {
-    cat >> $FLINK_DIR/conf/log4j.properties <<END
-rootLogger.appenderRef.console.ref = ConsoleAppender
-
-# Log all infos to the console
-appender.console.name = ConsoleAppender
-appender.console.type = CONSOLE
-appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%t] %-60c %x - %m%n
-END
-}
```

Comment:
Why don't we need this logging anymore?

Reply:
First, `setConsoleLogging` is not used in any native K8s related test scripts. Second, the native K8s is using `flink-console.properties` from release 1.12, which will output logs to console and files simultaneously.

Reply:
I noticed that we don't log any information when running `KubernetesLeaderElectionAndRetrievalITCase`. Could it be that we are using `log4j-test.properties` when running these tests because we use maven? Maybe we should set the log level to `INFO` in order to see the logs. Alternatively, we would have to use a log4j file which configures a file for the output which is then uploaded when the e2e fail (similarly to what we do when running the unit/integration tests which write into `mvn-1.log` and `mvn-2.log`. I think we use https://github.com/apache/flink/blob/master/tools/ci/log4j.properties.

Maybe we should exactly do this and use https://github.com/apache/flink/blob/master/tools/ci/log4j.properties for the logging configuration for the e2e tests which use maven to run them.

Reply:
I will have a look on the logging of unit test and try to correct the logging configuration for e2e tests that use `run_mvn`.